pages/linux/kdialog.md

```diff
@@ -0,0 +1,36 @@
+# kdialog
+
+> Show KDE dialog boxes from within shell scripts.
+> More information: <https://develop.kde.org/deploy/kdialog/>.
+
+- Open a dialog box displaying a custom message:
+
+`kdialog --msgbox "{{message}}" "{{optional_detailed_message}}"`
+
+- Open a question dialog with a `yes` and `no` button, returning `0` and `1`, respectively:
+
+`kdialog --yesno "{{message}}"`
+
+- Open a warning dialog with a `yes`, `no`, and `cancel` button, returning `0`, `1`, or `2` respectively:
+
+`kdialog --warningyesnocancel "{{message}}"`
+
+- Open an input dialog box and print the input to stdout when `OK` is pressed:
+
+`kdialog --inputbox "{{message}}" "{{optional_default_text}}"`
+
+- Open a dialog to prompt for a password and print it to stdout:
```

Comment:
```suggestion
- Open a dialog to prompt for a specific password and print it to stdout:
```

Reply:
I'm not so sure about this one. It isn't really a "specific password", it's just "any password".

Reply:
It's like GPG, the prompt ask for a password for a specific service, or `sudo`. So it asks for a specific password.